pages/pil/dashboard/index.js

```diff
@@ -99,6 +120,7 @@ module.exports = settings => {
   app.get('/success', (req, res, next) => {
     success({
       licence: 'pil',
+      type: get(req.model, 'openTasks[0].data.action') === 'transfer' ? 'transfer' : null,
```

Comment:
What's this for?

Reply:
Or more specifically, why is it important to send specifically `null` in the negative case and not just the task action all the time?

Reply:
The success router handles fallback for application vs amendment if you don't pass it a value, didn't want to replicate that logic here.

Reply:
It feels like we're spreading that logic around, and possibly the logic to determine the type of the thing should be handled all in one place.